pkg/internal/checkup/checkup_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"net"
 	"strings"
 	"testing"
+	"time"
```

Comment:
Could you please give a few words in the commit message about the added test?

Reply:
DONE